### PR TITLE
Create java_fibonacci.java

### PR DESCRIPTION
**Pull request by Red-0111**

### CODE REVIEW

File: java_fibonacci.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+public class Fibonacci {
```
Comment (fharookshaik) [inline java_fibonacci.java:1]: Codacy found an issue: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6289366)

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- java_fibonacci.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6289366&bid=20462418)

File: java_fibonacci.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    public static void main(String[] args) {
+
+        int n = 100, t1 = 0, t2 = 1;
```
Comment (fharookshaik) [inline java_fibonacci.java:5]: Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6289366)

**Comment by Red-0111** [comment]
Please review

**Comment by Red-0111** [comment]
i dont understand why these errors are shown?

**Review by sudalaipandian — APPROVED** [review]
public class Fibonacci {
public static void main(String arg[]){
	int a =1,c,b=1;
	system.out.println("0 1 1");
	for (int i=0;i<10;i++){
			c = a + b; 
            a = b; 
            b = c; 
			system.out.println(c);
		}
	}
}

**Comment by Red-0111** [comment]
Is my pr approved?
Will it count towards hacktoberfest?

**Comment by fharookshaik** [comment]
Hi @Red-0111 . Some changes have been requested before accepting your PR. Go through them. For more detailed description tap on the links provided at the errors. 

Once you're done with the changes, your PR will be reviewed and accepted.  

Happy Coding!!
Happy Contributing!!

**Comment by fharookshaik** [comment]
Code already in that programming language. You can develop the existing code further. Read the contribution file for further details

**Pull request closed without merging**